wear/build.gradle

```diff
@@ -11,6 +11,7 @@ apply plugin: 'com.google.gms.google-services'
 apply from: "../base.gradle"
 
 android {
+    namespace 'au.com.shiftyjelly.pocketcasts'
```

Comment:
The Android Studio upgrade also made this change. All the other modules and apps already have this defined in their build.gradle file.